src/tasks.py

```diff
@@ -16,29 +16,135 @@
 
 from openrelik_worker_common.file_utils import create_output_file
 from openrelik_worker_common.task_utils import create_task_result, get_input_files
+from .text_formatter import heading4,heading5,bullet
```

Comment:
In the openrelik-worker-common lib we have markdown report functionality already. Please take a look here: https://github.com/openrelik/openrelik-worker-common/blob/main/openrelik_worker_common/reporting.py

And an example of using it in another worker: https://github.com/openrelik/openrelik-worker-analyzer-config/blob/609f5cebb440cb01d56d19d892bf8408f39a6b6f/src/analyzers/sshd_analyzer.py#L32

Let's use that instead to stay consitent and also, if you create a "task report" using it you will get some UI things for free as well :)